CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,97 @@
+# Code of Conduct
+
+Этот документ содержит правила и рекомендации, позволяющие обеспечить эффективную, позитивную и вдохновляющую атмосферу для участников сообщества.
+
+## Правила поведения
+
+Правила поведения соблюдаются всеми участниками, а также пользователями, взаимодействующими с участниками и сервисами сообщества.
+
+Правила поведения представляют собой краткое изложение ценностей и «здравого смысла» сообщества:
+  - Мы стремимся обеспечить дружественную, безопасную и гостеприимную среду для всех,
+  независимо от уровня опыта, гендерной идентичности и самовыражения, сексуальной ориентации,
+  инвалидности, внешнего вида, размера тела, расы, этнической принадлежности, возраста, религии,
+  национальности или других подобных характеристик;
+  - Избегайте использования откровенно сексуальных псевдонимов или других псевдонимов, которые могут
+для всех ухудшить дружескую, безопасную и гостеприимную среду;
+  - Будьте добры и вежливы;
+  - Уважайте других и примите тот факт, что люди расходятся во мнениях;
+  - Соблюдайте [правила общения в Slack](SLACK_RULES.md);
+  - Сведите к минимуму неконструктивную критику;
+     Конструктивное общение - это такое взаимодействие людей, при котором собеседники способны
+     услышать и понять мнение друг друга, получая для себя что-то полезное, возможно даже ценное.
+     Также это способность выразить свои мысли без оценок, не приписывая личной точки зрения к
+     поведению другого человека.
+  
+     Конструктивное общение состоит из:
+       - умения слушать;
+       - способности понять смысл услышанного;
+       - способности конструктивно разрешать возникающие конфликты, а еще лучше не допускать их возникновения;
+       - желания получать новую информацию, знания;
+       - доброжелательности, приветливости;
+       - способности аргументировано и спокойно отстаивать свою точку зрения;
+       - способности понять, почему оппонент не согласен с Вашим мнением;
+       - умение не навязывать свое мнение в негативной форме, подавляя личность другого человека;
+       - Категорическое непринятие позиции другого человека без конструктивных
+  аргументов расценивается как неуважение.
+  - Поддерживайте других и принимайте поддержку. Также благодарите участников за предоставленную помощь;
+  - Мы исключим вас из взаимодействия, если вы оскорбляете, унижаете или оскорбляете кого-либо.
+    Это не приветствуется. Мы понимаем термин «домогательства», как определяется в Citizen Code of Conduct.
+    Если у вас нет ясности относительно того, что может быть включено в эту концепцию, пожалуйста,
+    прочитайте их определение;
+  - Категорически не приветствуются спам, троллинг, флейм, травля или другие виды кражи внимания;
+  - Размещение рекламных и партнерских материалов не приветствуется. Мы делаем исключения для
```

Comment:
Should we add an explicit note about posting vacancies and corresponding #jobs channel?